src/Tasks/SubTree/Quiz.hs

```diff
@@ -33,5 +33,7 @@ generateSubTreeInst SubTreeConfig {syntaxTreeConfig = SynTreeConfig {..}, ..} =
       { tree
       , minInputTrees = minSubTrees
       , correctFormulas = Data.Set.map display correctTrees
+      , correctTrees = correctTrees
```

Comment:
One might argue that it is not necessary to include both `correctFormulas` and `correctTrees` in the instance type. Where/how often is `correctFormulas` used? Can it reasonably be replaced with `Data.Set.map display correctTrees` there?

Reply:
`correctFormulas` was only used in one place. I removed it and replaced its occurrences with `Data.Set.map display correctTrees`